rs/sns/governance/canister/governance_test.did

```diff
@@ -677,6 +678,40 @@ type RegisterDappCanisters = record {
   canister_ids : vec principal;
 };
 
+// This type is equivalant to `ICRC3Value`, but we give it another name since it is used here not
+// in the context of the ICRC-3 ledger standard. The justification is the same: The candid format
+// supports sharing information even when the client and the server involved do not have the same
+// schema (see the Upgrading and subtyping section of the candid spec). While this mechanism allows
+// to evolve services and clients independently without breaking them, it also means that a client
+// may not receive all the information that the server is sending, e.g. in case the client schema
+// lacks some fields that the server schema has.
+//
+// This loss of information is not an option for SNS voters deciding if an extension with particular
+// init args should be installed or if an extension function with particular arguments should be
+// called. The client must receive the same exact data the server sent in order to verify it.
```

Comment:
I'm wondering if this lack of definite shared schema makes it possible to deceive voters by putting fields into the config that are irrelevant to the actual installed extension?

Reply:
The supported schema will be an observable property of a supported extension type. (That part of the API will be implemented in the follow-up PR.)

Since the schema will be validated at proposal submission time, there's no risk involved for the voters.